api/capi/doc/nnstreamer_capi.md

```diff
@@ -0,0 +1,315 @@
+# nnstreamer capi
+
+
+An application which require a data stream pipeline with deep learning model filter can be constructed and assembled easily with nnstreamer c-api.
+
+The main features of nnstreamer-capi include:
+
+- **Construction of data pipeline based on [gstreamer](https://gstreamer.freedesktop.org/)**
+
+  You can compose the data stream pipeline through nnstreamer-capi with various elements which included at gstreamer.
+
+- **[Single-shot](#singleshotapi) & [Pipeline](#pipelineapi) API**
```

Comment:
Let's say Single instead of Single-shot

There are a few residue of the name "Single-shot" in headers, but many documents no longer use this old name.

Reply:
it's applied